kyo-data/shared/src/main/scala/kyo/Chunk.scala

```diff
@@ -135,7 +135,7 @@ sealed abstract class Chunk[A] extends Seq[A] derives CanEqual:
                 case c: Indexed[A] => c
                 case _             => Compact(toArrayInternal)
 
-    final def flatten[B](using ev: A =:= Chunk[B]): Chunk[B] =
+    final def flattenChunk[B](using ev: A =:= Chunk[B]): Chunk[B] =
```

Comment:
@hearnadam I also hit that issue with `flatten` not working if the nested collection isn't a `Chunk`.